common/lib/xmodule/xmodule/html_module.py

```diff
@@ -368,6 +368,13 @@ class StaticTabFields(object):
         scope=Scope.settings,
         default="Empty",
     )
+    instructors_only = Boolean(
+        display_name=_("Hide Page From Students"),
+        help=_("If this option is selected, only users with the instructor role will see this page"
```

Comment:
This isn't true-- you are limiting to course staff. "Instructor" (at least in the code) means "Admin".

> Course team members with the Staff role are course co-authors. They have full writing and editing privileges on all course content.

![image](https://cloud.githubusercontent.com/assets/484484/19092050/08898352-8a53-11e6-8bea-c05fa3225075.png)